src/components/layout/footer.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Container } from '../containers'
+import { Mobile } from '../containers/show'
```

Comment:
Can import it  directly from containers

```suggestion
import { Mobile } from 'components/containers'
```